src/index.js

```diff
@@ -367,9 +367,16 @@ function applyClassName({ attributes }) {
 
 
 function extend(base, props) {
```

Comment:
is `props` no longer used? and is there a reason not to use `...props` rather than `arguments`?